documentation/src/main/asciidoc/user_guide/eviction.adoc

```diff
@@ -190,16 +190,20 @@ Max idle in a cluster is quite a bit more complicated. Since each node can have
 
 This check is done when a given node finds an entry has expired via maxIdle. It then will ask all the other owners what their last access time was (if it hasn't expired). If all owners say that the entry has expired, then it will not be returned to the user and removed based on the operation.
 
+When an entry is found to have expired due to max idle in the expiration reaper, instead of updating the last access time to current time, it will update all nodes to the most recent access of all owners.
```

Comment:
I think the "updating the last access time to current time" bit is better suited in the paragraph above, and/or this paragraph should be moved after line 199

Reply:
Sure

Reply:
Moved to 199